### PR TITLE
IN-2775: execute_query_with_http_info() response fix

### PR DESCRIPTION
**Pull request by ddw-mike**
https://dataworld.atlassian.net/browse/IN-2775
Changes the response type of `execute_query_with_http_info()` to `PaginatedQueryResults` instead of `none`

### CODE REVIEW

File: datadotworld/client/_swagger/apis/queries_api.py [hunk 1]
```diff
@@ -632,7 +632,7 @@ def execute_query_with_http_info(self, id, **kwargs):
                                         body=body_params,
                                         post_params=form_params,
                                         files=local_var_files,
-                                        response_type=None,
+                                        response_type='PaginatedQueryResults',
```
Comment (jonaf) [inline datadotworld/client/_swagger/apis/queries_api.py:635]: I have no context to review this 😅 but, I think the api response type depends on the request in this case, no?

**Comment by ddw-mike** [comment]
@jonaf Does `swagger-dwapi-def.json` need to be updated here?

File: datadotworld/client/_swagger/apis/queries_api.py (hunk 1 above)
Comment (ddw-mike) [reply datadotworld/client/_swagger/apis/queries_api.py:635]: Shoot, didn't link the ticket
https://dataworld.atlassian.net/browse/IN-2775

**Comment by nikhilhegde-ddw** [comment]
@ddw-mike Please see my comment in intake.

**Comment by jonaf** [comment]
Changing return type to `object` seems fine. It's basically saying "return whatever" I suppose.. I do agree that updating the swagger definition would be appropriate _iff_ we knew how best to account for the response types, but I don't even think there's an appropriate _correct_ version of Swagger (that we have or know of today) that correctly accounts for the range of valid responses for this endpoint. So this seems okay as a workaround to me.

**Review by ChiragThesia — APPROVED** [review]
Per Mike's requests and based on Jona's comment approving this so we can unblock the customer.

**Pull request merged**